src/components/MindmapWrapper.vue

```diff
@@ -105,26 +105,31 @@ export default {
   watch: {
     items: {
       async handler() {
-        this.itemLists = this.items;
+        this.itemLists = structuredClone(this.items);
         let newMap = { ...this.emojiMenu };
         this.itemLists.map(async (item) => {
+          newMap[`menu-${item.stepID}`] = false;
           let temp = structuredClone(item);
-          newMap[`menu-${temp.stepID}`] = false;
-          if (this.getType(temp.fileType) === "mindmap")
-            temp.data = await this.$storageService.getItemById(temp.stepID);
+          if (this.getType(item.fileType) === "mindmap") {
+            if (this.$isElectron)
+              temp.data = await this.$storageService.getItemById(temp.stepID);
+            else {
+              const itemInStore = this.$store.state.session.items.find(
```

Comment:
just fix `src/services/storage-options/restApiService.js:getItemById` and then you don't have to manually access the store here

`getItemById` actually seems to be a bit confused....... maybe it should always be pulling the item from the store instead of from the JSON state or API.  please look into this function and how it is used to decide the best way to do this consistently